test/zcl_abapgit_tadir_test.clas.testclasses.abap

```diff
@@ -0,0 +1,169 @@
+CLASS ltcl_build DEFINITION DEFERRED.
+
+CLASS ltcl_build DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION SHORT FINAL.
```

Comment:
its not deferred, its defined in the next statement, MEDIUM runtime will make it execute later than the tests in /src/

```suggestion
CLASS ltcl_build DEFINITION FOR TESTING RISK LEVEL HARMLESS DURATION MEDIUM FINAL.
```

Reply:
I adjusted it, but now the unit test failed and I don't know why. In Eclipse in the src folder was it ok.
File \test\zcl_abapgit_tadir_test.clas.testclasses.abap

Reply:
<img width="858" alt="image" src="https://github.com/abapGit/abapGit/assets/101019006/93703b64-e365-49ee-9f01-28a5011095af">

Reply:
fixed with https://github.com/abapGit/abapGit/pull/6429#issuecomment-1691678003